superset/migrations/versions/b8d3a24d9131_new_dataset_models.py

```diff
@@ -292,78 +305,70 @@ def after_insert(target: SqlaTable) -> None:  # pylint: disable=too-many-locals
         columns.append(
             NewColumn(
                 name=metric.metric_name,
-                type="Unknown",  # figuring this out would require a type inferrer
-                expression=metric.expression,
-                warning_text=metric.warning_text,
                 description=metric.description,
+                expression=metric.expression,
```

Comment:
Oh my. I do love me some ABC.

Reply:
I know this would make you happy, John! But unfortunately I think this block will be eventually removed once we migrate to INSERT FROM SELECT.